### PR TITLE
Started adding outcome buttons to issue contact detail

### PR DESCRIPTION
**Pull request by nickoneill**
Created some outcome buttons, connected them to reporting API call. Interestingly once I moved the buttons into another view, I had to manually push new items into the nav stack, but that was relatively easy with the router.

https://github.com/5calls/ios/assets/49688/3382ac28-03b6-4f3d-a385-12189b343d80

### CODE REVIEW

File: FiveCalls/FiveCalls.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -0,0 +1,158 @@
+{
```
Comment (cselin) [inline FiveCalls/FiveCalls.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: is this file change intentional?

**Review by cselin — APPROVED** [review]
Looks good, screenshot for final changes?

File: FiveCalls/FiveCalls.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (nickoneill) [reply FiveCalls/FiveCalls.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: Yep, must have been a Xcode 15 change, I keep seeing it pop up in other branches

**Comment by nickoneill** [comment]
OK, really ready now :) I hooked up the API call for reporting too

File: FiveCalls/FiveCalls/IssueContactDetail.swift [hunk 2]
```diff
@@ -60,26 +62,29 @@ struct IssueContactDetail: View {
...
-                    .padding(.horizontal)
+                    .padding(.bottom)
                 if remainingContacts.count > 1 {
                     NavigationLink(value: IssueDetailNavModel(issue: issue, contacts: nextContacts)) {
-                        PrimaryButton(title: R.string.localizable.nextContact(),
-                                      systemImageName: "megaphone.fill")
-                        .padding()
+                        OutcomesView(outcomes: issue.outcomeModels, report: { outcome in
+                            let log = ContactLog(issueId: String(issue.id), contactId: currentContact.id, phone: "", outcome: outcome.status, date: Date(), reported: true)
+                            store.dispatch(action: .ReportOutcome(log, outcome))
```
Comment (cselin) [inline FiveCalls/FiveCalls/IssueContactDetail.swift:72]: At first it seemed a little odd to me to pass a `report` closure into the `OutcomesView`, but I guess it avoids having to expose the issue, currentContact, etc to the sub view.

**Review by cselin — APPROVED** [review]

**Pull request merged**